### DISCUSSION TITLE
"Hide" vs. Remove zap functionality

### DISCUSSION THREAD

**Discussion by jgriegershs** [Siemens-Healthineers/K2s#177] [Ideas]
Users keep using the zap functionality and report messed-up systems.

So how do we deal with this? Remove the functionality completely, "hide" it or show at least warnings about the risks and ask for additional confirmation?

**Comment by abhitoshj** [comment]
The k2s image reset-win-storage command is quite slow but more reliable. Since it is quite slow, users sometimes tend to use zap.exe directly to delete the folder faster.

As part of the implementation of issue #31, we propose to add a message to the user indicating that the deletion will take quite some time and the command asks for a confirmation.

**Reply by abhitoshj** [reply]
But the real issue for me, is in windows image handling of containerd. Even though, the user has deleted all windows images, there are many dangling layers/files left in the image cache. This cache grows over a period and becomes so big that k2s image reset-win-storage takes a lot of time to delete this cache.
